### PR TITLE
Bento release 4.0.0 feature gmb 68 files cart

### PR DESCRIPTION
**Pull request by huuaho**
## Description

Files can now be added from the explore and subject details pages

Fixes # GMB-68

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

### CODE REVIEW

File: src/bento/dashboardTabData.js [hunk 1]
```diff
@@ -75,7 +75,7 @@ export const tabIndex = [
 ];
 
 export const DASHBOARD_QUERY_NEW = gql`
-query search (
+query search1 (
```
Comment (iksheth) [inline src/bento/dashboardTabData.js]: @huuaho Is it possible to call query-only "search"? Is there any specific reason we need to have search1, Search2, and so on..

**Review by iksheth — CHANGES_REQUESTED** [review]

File: src/bento/dashboardTabData.js (hunk 1 above)
Comment (huuaho) [reply src/bento/dashboardTabData.js]: it was for debugging, and if i wanted to figure out which "search" query was throwing an error, the numbers are now removed.

Comment (iksheth) [reply src/bento/dashboardTabData.js]: @huuaho I can not see any changes or any new commit.

Comment (huuaho) [reply src/bento/dashboardTabData.js]: apologies, did not push to repo

**Review by iksheth — APPROVED** [review]

**Pull request merged**